src/xdr/Stellar-transaction.x

```diff
@@ -378,6 +379,24 @@ struct ClawbackOp
     int64 amount;
 };
 
+/* SetTrustLineFlagsOp
+
+   Updates the flags of an existing trust line.
+   This is called by the issuer of the related asset.
+
+   Threshold: low
+
+   Result: SetTrustLineFlagsResult
+*/
+struct SetTrustLineFlagsOp
+{
+    AccountID trustor;
+    Asset asset;
+
+    uint32* clearFlags; // which flags to clear
```

Comment:
This was done to match the existing `SetOptionsOp` parameters.

Reply:
@MonsieurNicolas does make an interesting observation here. Just because `SetOptionsOp` has a slightly strange interface doesn't mean that we have to propagate that to `SetTrustLineFlagsOp`. I know @leighmcculloch has a lot of thoughts about interface consistency, so I'm interested to hear his take on this.

Reply:
I don't think this has any meaningful impact one way or the other. Looks like you could drop the pointer to me.

Reply:
Yeah I don't feel strongly about either approach. If any of you do think we should drop the pointer, I'm fine with that. Let me know and I'll update this PR and the CAP.